src/lazy/binary/raw/v1_1/e_expression.rs

```diff
@@ -141,7 +141,7 @@ impl<'top> RawEExpression<'top, v1_1::Binary> for &'top BinaryEExpression_1_1<'t
     type ArgGroup = BinaryEExpArgGroup<'top>;
 
     fn id(self) -> MacroIdRef<'top> {
-        MacroIdRef::LocalAddress(self.macro_ref.address())
+        self.macro_ref.id()
```

Comment:
🗺️ Throughout this patch, places using a "macro address" have been generalized to work with a "macro ID," allowing them to distinguish between an address in the application's macro table and one in the system table.